### PR TITLE
feat: source request submitted email

### PR DESCRIPTION
**Pull request by rebelchris**
Introducing a change from the old mailer function to include the source request mail in the API.

Small question:
I thought there was another place where we had to introduce these topics, am I mistaken?

Note:
We also might want to move 
- decline
- approve 

emails to be in the API so we can close the old mailer function

WT-251 #done

### CODE REVIEW

**Review by kirillkurko — APPROVED** [review]

**Review by sshanzel — APPROVED** [review]
Nice! I think you added the pubsub topic in the right place.

**Comment by idoshamun** [comment]
@rebelchris, I agree that it is better to migrate the mailer functionality completely to the API.
I don't think we need to introduce a new topic here, as the mailer already subscribes to existing topics.
We need to add new subscribers to them and then delete the mailer.

**Comment by rebelchris** [comment]
@idoshamun I kept the integration the same so the one file can send multiple mailers related to the source requests.
This way we keep the same topic and can remove the mailer from the daily-functions repo.

File: src/workers/index.ts [hunk 1]
```diff
@@ -30,6 +30,7 @@ import usernameChanged from './usernameChanged';
 import updateComments from './updateComments';
 import deleteUser from './deleteUser';
 import cdc from './cdc';
+import sourceRequestSubmittedMail from './sourceRequestMail';
```
Comment (idoshamun) [inline src/workers/index.ts:33]:
```suggestion
import sourceRequestMail from './sourceRequestMail';
```

File: src/workers/index.ts [hunk 2]
```diff
@@ -47,6 +48,7 @@ export const workers: Worker[] = [
   postScoutMatchedMail,
   communityLinkAccessMail,
   communityLinkRejectedMail,
+  sourceRequestSubmittedMail,
```
Comment (idoshamun) [inline src/workers/index.ts:51]:
```suggestion
  sourceRequestMail,
```

**Review by idoshamun — APPROVED** [review]
I usually don't like to introduce breaking changes to a pubsub message, but in this case it might be fine. Please make sure that no one else beside the mailer subscribes to the topic before you merge it. You can use GCP UI for that

**Comment by rebelchris** [comment]
@idoshamun for your eyes, I revamped the source request reputation subscription to reflect the new flow.

**Review by idoshamun — APPROVED** [review]

**Pull request merged**